viewer/core/src/glsl/math/floatBitsSubset.glsl

```diff
@@ -1,17 +1,9 @@
-const int MAX_ITER = 8;
-
 float floatBitsSubset(float inNumber, int fromLeastSignificantBitIndex,  int toMostSignificantBitIndex) {
-    float lsbif = float(fromLeastSignificantBitIndex);
-    int a = int(toMostSignificantBitIndex - fromLeastSignificantBitIndex);
-    float denominator = pow(2.0, lsbif);
-    float outNumber = 0.0;
-    for(int i = 0; i < MAX_ITER; i++)
-    {
-      if(i >= a) break;
-      float backBits = pow(2.0, lsbif + float(i));
-      outNumber += (mod(inNumber, backBits * 2.0) - mod(inNumber, backBits)) / denominator;
-    }
-    return outNumber;
+    float r = float(fromLeastSignificantBitIndex);
+    float l = float(toMostSignificantBitIndex);
+
+    float bitShift = pow(2.0, r);
+    return mod(((inNumber - mod(inNumber, bitShift)) / bitShift), pow(2.0, l - r));

```

Comment:
Very cool, but I think it would be good if you doc this with the example you showed